scripts/update-changelog.ts

```diff
@@ -0,0 +1,78 @@
+import { execSync } from 'node:child_process'
+import { $fetch } from 'ofetch'
+import { inc } from 'semver'
+import { generateMarkDown, getCurrentGitBranch, loadChangelogConfig } from 'changelogen'
+import { consola } from 'consola'
+import { determineBumpType, getLatestCommits, loadWorkspace } from './_utils'
+
+async function main () {
+  const releaseBranch = await getCurrentGitBranch()
+  const workspace = await loadWorkspace(process.cwd())
+  const config = await loadChangelogConfig(process.cwd(), {})
+
+  const commits = await getLatestCommits().then(commits => commits.filter(
+    c => config.types[c.type] && !(c.type === 'chore' && c.scope === 'deps' && !c.isBreaking)
+  ))
+  const bumpType = await determineBumpType()
+
+  const newVersion = inc(workspace.find('@nuxt/bridge').data.version, bumpType || 'patch')
```

Comment:
I think we might need to force this to be an RC bump for now or it will create a release pr rather than an RC pr.

Reply:
Something like this: 
https://github.com/nuxt/bridge/pull/941/commits/e0acd4fc77e90258e59e630c2fd499e325b71cf1